Description: some tests fail on mips64el and mipsel
 That architecture support improves over time - node 20.x branch has better support for mips64el
 Meanwhile, let those tests fail.
Forwarded: not-needed
--- nodejs-18.19.0+dfsg.orig/test/parallel/parallel.status
+++ nodejs-18.19.0+dfsg/test/parallel/parallel.status
@@ -97,6 +97,19 @@ test-wasm-web-api: SKIP
 [$arch==mips64el]
 # the debug flag is for hacking v8 internals
 test-debug-args: PASS,FLAKY
+test-cli-node-options: SKIP
+test-crypto-sign-verify: PASS,FLAKY
+test-freeze-intrinsics: PASS,FLAKY
+test-http2-forget-closed-streams: PASS,FLAKY
+test-tls-root-certificates: PASS,FLAKY
+test-vm-global-non-writable-properties: PASS,FLAKY
+test-vm-global-setter: PASS,FLAKY
+test-vm-module-synthetic: PASS,FLAKY
+test-vm-strict-assign: PASS,FLAKY
+
+[$arch==mipsel]
+test-cli-node-options: SKIP
+test-debugger-extract-function-name: SKIP
 
 [$system==solaris] # Also applies to SmartOS
 # https://github.com/nodejs/node/issues/43457
